### ISSUE TITLE
Running GUI

### ISSUE THREAD

**Issue by librarianmage** [mxgmn/MarkovJunior#11]
Hello,

I can't seem to figure out how to run the GUI. The code seems to be included in the source, but I haven't a clue on how to get it to start.

**Comment by Magnogen** [comment]
I've tried running the .exe in the release too, but it doesn't seem to do anything. Could it perhaps need the commandline?

**Comment by inkydragon** [comment]
This may be a naming confusion.

The `GUI` parameter is actually used to specify the width of the sidebar for rules in the output image.


Just modify your `models.xml`:

- `<model name="Circuit" size="100" steps="1200"/>`
![image](https://user-images.githubusercontent.com/5158738/171793237-beeac397-dc57-436c-83b6-70fc9d8bded1.png)


- `<model name="Circuit" size="100" steps="1200" gui="200"/>`
![image](https://user-images.githubusercontent.com/5158738/171792998-364d5e63-afc7-4aec-8886-44582e7182fb.png)

**Comment by nonchip** [comment]
> This may be a naming confusion.

agreed, if it's not an interface, it can't be a user interface. i'd call that more of a "state graph" or something.

**Comment by Magnogen** [comment]
In which case, how possible is it to make a UI that can hold rules and compile to xml and then draw what it already has?

**Comment by Magnogen** [comment]
Ahhh, alright. Thank you for the clarification

**Comment by gabrielelibardi** [comment]
Hi, I am running MJ on Ubuntu from the console with `dotnet run --configuration Release MarkovJunior.csproj`. It looks like it is running fine after a fix from [here](https://github.com/mxgmn/MarkovJunior/pull/4) but it is not showing anything. My goal would be to see the program as it is being executed as in the gifs. I also tried to replace the line `<model name="Circuit" size="100" steps="1200"/>` to no avail. Is there anything really obvious I am missing? I am new to C#

**Comment by mxgmn** [comment]
@gabrielelibardi add `gif="True"` to this line:
```xml
<model name="Circuit" size="100" steps="1200" gui="300" gif="True"/>
```

**Comment by gabrielelibardi** [comment]
That works thanks!

**Comment by tigrouind** [comment]
So it's not a real-time app but actually produce a gif

**Comment by mxgmn** [comment]
It produces animation frames that you can then convert into a gif.
